src/main/app/claims/claimStoreClient.ts

```diff
@@ -45,6 +47,27 @@ export class ClaimStoreClient {
       }
     })
   }
+
+  static acceptOfferByUser (acceptedBy: string, user: User): Promise<Claim> {
```

Comment:
Would be nice to rename methods in this class to have just `makeOffer`, acceptOffer` and `rejectOffer`.

Reply:
What do you think about having a separate client for it as well?

Reply:
I was thinking about it as we have a separate client for CCJ as well.